### DISCUSSION TITLE
Unable to create a chroot

### DISCUSSION THREAD

**Discussion by mt025** [ptitSeb/box86#317] [Q&A]
I have a build the latest build of box86 and a 32-bit debian stretch chroot, but I can't get them to work. I'm pretty sure its possible as I've seen people talking about it, but I hope I'm not mistaken. I know I don't need the chroot, but I am looking to test out some things in a full 32-bit env.

Seems to work fine on QEMU, when copying the static binary to the chroot and running `update-binfmts --enable qemu-i386` on the normal system to swap from box86 to QEMU.

I have done the same with box86 and copied the build files over to `/usr/local/bin` and `/usr/lib/i386-linux-gnu/`. I just get the error "No such file or directory", which I have read could be due to missing libraries, even though it says the file is missing when its not. Do I need to include arm libs in my chroot? What am I missing with box86?

```
pi@J-Pi:~/scripts/Builds/bootstrap$ sudo chroot ./chroot-stretch-i386/
chroot: failed to run command ‘/bin/bash’: No such file or directory

pi@J-Pi:~/scripts/Builds/bootstrap$ ls ./chroot-stretch-i386/bin/bash
./chroot-stretch-i386/bin/bash

```

**Comment by mbodomi** [comment]
Try to use schroot!
